WhaleSpotting/Repositories/WhaleSightingRepo.cs

```diff
@@ -32,4 +33,12 @@ public WhaleSighting GetById(int id)
             throw new ArgumentOutOfRangeException($"No sightning with id {id} found in the database", ex);
         }
     }
+    public List<WhaleSighting> GetPendingSightings()
+    {
+        return context.WhaleSightings
+        .Where(ws => ws.ApprovalStatus == ApprovalStatus.Pending)
+        .Include(ws => ws.User)
+        .Include(ws => ws.WhaleSpecies)
+        .ToList();
```

Comment:
We can add a `.Select()` in here to return these as a `WhaleSightingResponse`
```suggestion
        .Where(ws => ws.ApprovalStatus == ApprovalStatus.Pending)
        .Include(ws => ws.User)
        .Include(ws => ws.WhaleSpecies)
        .Select(ws => new WhaleSightingResponse(ws))
        .ToList();
```